src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -372,4 +372,14 @@ public boolean isSectionChanged(StudentAttributes originalStudentAttribute) {
     public boolean isTeamChanged(StudentAttributes originalStudentAttribute) {
         return this.team != null && !this.team.equals(originalStudentAttribute.team);
     }
+    
+    /**
+     * Checks whether the edit form of student has changed the email value.
+     * 
+     * @param originalStudentAttribute
```

Comment:
If there is nothing else to say about the parameter, remove the line altogether.